sources/core/objects/Drinks.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Drinks.h"
+
+Drinks::Drinks(quint32 id) : Item(id)
+{
+    name = "BaseDrink";
+    SetSprite("icons/drinks.dmi");
+    full_ = false;
+}
+void Drinks::Drink(IdPtr<Human> human1, IdPtr<Human> human2)
```

Comment:
It is better to name humans more meaningful. Something like `consumer` and `feeder`.